text/0097-log-data-model.md

```diff
@@ -0,0 +1,971 @@
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>host</td>
+    <td>string</td>
+    <td>The host value to assign to the event data. This is typically the host name of the client that you are sending data from.</td>
+    <td>resource["host.hostname"]</td>
+  </tr>
+  <tr>
+    <td>source</td>
+    <td>string</td>
+    <td>The source value to assign to the event data. For example, if you are sending data from an app you are developing, you could set this key to the name of the app.</td>
+    <td>resource["service.name"]</td>
+  </tr>
+  <tr>
+    <td>sourcetype</td>
+    <td>string</td>
+    <td>The sourcetype value to assign to the event data.</td>
+    <td>attributes["source.type"]</td>
+  </tr>
+  <tr>
+    <td>event</td>
+    <td>any</td>
+    <td>The JSON representation of the raw body of the event. It can be a string, number, string array, number array, JSON object, or a JSON array.</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>fields</td>
+    <td>Map of any</td>
+    <td>Specifies a JSON object that contains explicit custom fields.</td>
+    <td>attributes</td>
+  </tr>
+  <tr>
+    <td>index</td>
+    <td>string</td>
+    <td>The name of the index by which the event data is to be indexed. The index you specify here must be within the list of allowed indexes if the token has the indexes parameter set.</td>
+    <td>TBD, most like will go to attributes</td>
+  </tr>
+</table>
+
+
+### Log4j
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>Instant</td>
+    <td>timestamp</td>
+    <td>Time when an event occurred measured by the origin clock.</td>
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td>Level</td>
+    <td>enum</td>
+    <td>Log level.</td>
+    <td>severity</td>
+  </tr>
+  <tr>
+    <td>Message</td>
+    <td>string</td>
+    <td>Human readable message.</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>any</td>
+    <td>Structured data.</td>
+    <td>attributes</td>
+  </tr>
+</table>
+
+
+### Zap
+
+<table>
+  <tr>
+    <td>Field</td>
+    <td>Type</td>
+    <td>Description</td>
+    <td>Maps to Unified Model Field</td>
+  </tr>
+  <tr>
+    <td>ts</td>
+    <td>timestamp</td>
+    <td>Time when an event occurred measured by the origin clock.</td>
+    <td>timestamp</td>
+  </tr>
+  <tr>
+    <td>level</td>
+    <td>enum</td>
+    <td>Logging level.</td>
+    <td>severity</td>
+  </tr>
+  <tr>
+    <td>caller</td>
+    <td>string</td>
+    <td>Calling function's filename and line number.
+</td>
+    <td>attributes, key=TBD</td>
+  </tr>
+  <tr>
+    <td>msg</td>
+    <td>string</td>
+    <td>Human readable message.</td>
+    <td>body</td>
+  </tr>
+  <tr>
+    <td>All other fields</td>
+    <td>any</td>
+    <td>Structured data.</td>
+    <td>attributes</td>
+  </tr>
+</table>
+
+
+### Apache
```

Comment:
```suggestion
### Apache HTTPD access log
```

I'm open to alternate wording, but "Apache" covers so many projects (including log4j) that I find myself wanting more-precise language.

Reply:
Looks like the official name is "Apache HTTP Server". I will use that.